packages/app/src/systems/Bridge/types.tsx

```diff
@@ -16,8 +16,8 @@ export type BridgeTx = {
 };
 
 export type BridgeAsset = {
-  address?: string;
+  address: string;
   decimals?: number;
-  symbol?: string;
+  symbol: string;
```

Comment:
should be all "?" as it was

Reply:
i'll change it back, but what is a scenario when address can undefined?  Even for eth we use that NativeAssetId